modules/integration/tests-common/admin-clients/src/main/java/org/wso2/identity/integration/common/clients/application/mgt/DefaultAuthSeqMgtServiceClient.java

```diff
@@ -0,0 +1,195 @@
+import org.apache.axis2.context.ConfigurationContext;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.identity.application.common.model.xsd.DefaultAuthenticationSequence;
+import org.wso2.carbon.identity.application.mgt.defaultsequence.stub.IdentityDefaultSeqManagementServiceDefaultAuthSeqMgtException;
+import org.wso2.carbon.identity.application.mgt.defaultsequence.stub.IdentityDefaultSeqManagementServiceStub;
+
+import java.rmi.RemoteException;
+
+/**
+ * Client for DefaultAuthSeqMgtService.
+ */
+public class DefaultAuthSeqMgtServiceClient {
+
+    private IdentityDefaultSeqManagementServiceStub stub;
+    private static final Log log = LogFactory.getLog(DefaultAuthSeqMgtServiceClient.class);
+    private final boolean debugEnabled = log.isErrorEnabled();
+    private static final String DEFAULT_AUTH_SEQ = "default_sequence";
+
+    public DefaultAuthSeqMgtServiceClient(String cookie, String backendServerURL,
+                                          ConfigurationContext configCtx) throws Exception {
+
+        String serviceURL = backendServerURL + "IdentityDefaultSeqManagementService";
+        stub = new IdentityDefaultSeqManagementServiceStub(configCtx, serviceURL);
+
+        ServiceClient client = stub._getServiceClient();
+        Options option = client.getOptions();
+        option.setManageSession(true);
+        option.setProperty(org.apache.axis2.transport.http.HTTPConstants.COOKIE_STRING, cookie);
+
+        if (debugEnabled) {
+            log.debug("Invoking service " + serviceURL);
+        }
+    }
+
+    /**
+     * Create default authentication sequence.
+     *
+     * @param authenticationSequence default authentication sequence
+     * @throws IdentityDefaultSeqManagementServiceDefaultAuthSeqMgtException
+     */
+    public void createDefaultAuthenticationSeq(DefaultAuthenticationSequence authenticationSequence)
+            throws IdentityDefaultSeqManagementServiceDefaultAuthSeqMgtException {
+
+        try {
+            if (debugEnabled) {
+                log.debug("Creating default authentication sequence.");
+            }
+            stub.createDefaultAuthenticationSeq(authenticationSequence);
+        } catch (RemoteException e) {
+            log.error("Error occurred when creating default authentication sequence.", e);
+            throw new IdentityDefaultSeqManagementServiceDefaultAuthSeqMgtException("Server error occurred.");
+        }
+    }
+
+    /**
+     * Retrieve default authentication sequence in XML.
+     *
+     * @return default authentication sequence in XML
+     * @throws IdentityDefaultSeqManagementServiceDefaultAuthSeqMgtException
+     */
+    public DefaultAuthenticationSequence getDefaultAuthenticationSeqInXML() throws
+            IdentityDefaultSeqManagementServiceDefaultAuthSeqMgtException {
+
+        try {
+            if (debugEnabled) {
+                log.debug("Retrieving default authentication sequence.");
```

Comment:
Shall we modify this to log.debug("Retrieving default authentication sequence in XML format");?
Because https://github.com/wso2/product-is/pull/3820/files#diff-b7c834677dcad17329d57380deab4ecfR109 also has the same log.